src/main/java/cat/udl/eps/entsoftarch/repository/TagRepository.java

```diff
@@ -13,4 +13,5 @@
 @RepositoryRestResource
 public interface TagRepository extends PagingAndSortingRepository<Tag, String> {
     List<Tag> findByName(@Param("name") String name);
+    List<Tag> findByNameContaining(@Param("name") String name);
```

Comment:
The feature is about looking for tags starting with some text. The implementation is about containing...